Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
>__CFISAForTypeID [](start = 10, length = 16)

I'm not totally happy that this was made non-inline. Is there a way you can preserve the inlined version for CF internal consumers and keep an out-of-line version for external ones?